src/main/kotlin/no/nav/amt/person/service/nav_ansatt/NavAnsattUpdater.kt

```diff
@@ -1,25 +1,50 @@
 package no.nav.amt.person.service.nav_ansatt
 
 import no.nav.amt.person.service.clients.nom.NomClient
+import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Component
+import java.util.UUID
 
 @Component
 class NavAnsattUpdater(
+	private val repository: NavAnsattRepository,
 	private val nomClient: NomClient,
-	private val navAnsattRepository: NavAnsattRepository,
 ) {
 
-	private val batchSize = 100
+	private val log = LoggerFactory.getLogger(javaClass)
 
-	fun oppdaterAlle() {
-		val ansattBatcher = navAnsattRepository.getAll()
-			.map { it.navIdent }
-			.chunked(batchSize)
+	fun oppdaterAlle(batchSize: Int = 100) {
+		val ansattBatcher = repository.getAll().chunked(batchSize)
 
 		ansattBatcher.forEach { batch ->
-			val oppdaterteAnsatte = nomClient.hentNavAnsatte(batch)
-			navAnsattRepository.updateMany(oppdaterteAnsatte)
+			val ansatte = batch.associate { it.navIdent to AnsattSomSkalOppdateres(it.id, false) }
```

Comment:
20+ ekstra linjer for å legge på logging og for å ikke bruke `NomNavAnsatt` i repository... 😅 

Godt mulig dette kan gjøres mye enklere.